stl/inc/type_traits

```diff
@@ -1660,13 +1660,14 @@ struct _Invoker_ret<_Unforced, false> { // selected for _Rx being _Unforced
 
 
 // TYPE TRAITS FOR invoke()
-template <class _To>
-void _Implicitly_convert_to(_To) noexcept; // not defined
-
 #pragma warning(push)
 #pragma warning(disable : 4242) // 'identifier': conversion from '_From' to '_To', possible loss of data (/Wall)
 #pragma warning(disable : 4244) // 'argument': conversion from '_From' to '_To', possible loss of data
 #pragma warning(disable : 4365) // 'argument': conversion from '_From' to '_To', signed/unsigned mismatch (/Wall)
+#pragma warning(disable : 5215) // '%s' a function parameter with a volatile qualified type is deprecated in C++20

```

Comment:
I find it somewhat interesting that you don't need to suppress Clang `-Wdeprecated-volatile` here. But if there are no warnings in our test suite, I'm happy.

Reply:
We probably should; this will be in some future set of "suppress clang warnings that don't show up because system header" 🤷‍♂ .